src/third/redux-persist/createPersistor.js

```diff
@@ -139,18 +143,6 @@ export default function createPersistor (store, config) {
   }
 }
 
-function warnIfSetError (updatedSubstates) {
-  return function setError (err) {
-    if (err) {
-      logging.warn(
```

Comment:
createPersistor [nfc]: Inline a complicated helper function we use once.
    
    This is the first commit in a series that will rework
    `createPersistor` to use `storage.multiSet` instead of many
    `storage.setItem`s, taking care not to detract from performance. For
    background, please see
      https://chat.zulip.org/#narrow/stream/243-mobile-team/topic/data.20storage/near/1214393.

nit: can drop that commit-message body, as this is now the last commit in the series instead of the first :slightly_smiling_face: